ui/oopsyraidsy/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -0,0 +1,51 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+import { playerDamageFields } from '../../../oopsy_common';
+
+export type Data = OopsyData;
+
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.TheMinstrelsBalladZodiarksFall,
+  damageWarn: {
+    'ZodiarkEx Quetzalcoatl Keraunos Eidolon': '67E1', // bird donut
+    'ZodiarkEx Behemoth Meteoros Eidolon': '67E2', // behemoth circle
+    'ZodiarkEx Python Opheos Eidolon': '67E3', // python line
+    'ZodiarkEx Infernal Stream 1': '52D1', // initial fire wall
+    'ZodiarkEx Infernal Stream 2': '67E0', // rotating fire wall
+    'ZodiarkEx Esoteric Ray': '67E4', // laser wall summon
+    'ZodiarkEx Esoteric Dyad': '67E5', // half arena wall summon
+    'ZodiarkEx Esoteric Sect': '67E6', // conal wall summon
+    'ZodiarkEx Algedon': '67EE', // untelegraphed diagonal knockback
+    'ZodiarkEx Adikia 1': '63A9', // two hit east west circles
+    'ZodiarkEx Adikia 2': '67F2', // two hit east west circles
+    'ZodiarkEx Explosion': '67E7', // Astral Eclipse explosions
```

Comment:
Is Explosion even survivable for non-tanks?

Reply:
Yeah, I think you're right that this should probably be fail and not warn.